android/app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +17,7 @@
 
       <meta-data
         android:name="com.google.android.geo.API_KEY"
-        android:value="AIzaSyAHaa3pnWi6NGPZb-sGawBW-kIf2jQH4QE"/>
+        android:value="AIzaSyAkTPaxSFLk2UJEf0IJd3exWzoAgd3DZPk"/>
```

Comment:
I think that change can have an impact on the production build. I doubt you want it in that PR.

Reply:
It's necessary, I pushed an old key that's not for our project, the new one was provided by Serge